### PR TITLE
{lib}[foss/2016a] libQGLViewer 2.6.3 (REVIEW)

### PR DESCRIPTION
**Pull request by jrbosch**
libQGLViewer easyconfig for foss
needs easyblock [hpcugent/easybuild-easyblocks#890 ](https://github.com/hpcugent/easybuild-easyblocks/pull/890)

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test (tests will fail while easyblock is not merged yet, but fine)

File: easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.6.3-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description = "libQGLViewer is a C++ library based on Qt that eases the creation of OpenGL 3D viewers."
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://www.libqglviewer.com/src/']
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [('Qt', '4.8.7')]
+
+start_dir = '%(builddir)s/libQGLViewer-2.6.3/QGLViewer'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.6.3-foss-2016a.eb]: don't hardcode the version here please, use `%(version)s` instead:

```python
start_dir = '%(builddir)s/libQGLViewer-%(version)s/QGLViewer'
```

File: easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.6.3-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+easyblock = 'EB_libQGLViewer'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.6.3-foss-2016a.eb]: no need to hardcode this, if no generic easyblock is specified, EasyBuild will automatically look for an easyblock that matches the software name; so, just remove this line (and the empty one below)

**Comment by jrbosch** [comment]
after include libgl as dependency, the problem of not finding GL/glu.h is solved. thanks @boegel

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d2a647d04533f914d57bb65d523ab47f for a full test report.

**Comment by boegel** [comment]
tested on top of https://github.com/hpcugent/easybuild-easyblocks/pull/890, works great now

@jrbosch please remove the hardcoded `easyblock` line, and it's good to go

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c7ae4193f6bb1c92beee4ddee6b718ff for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f6536de51ad6ae88e3c571b455b5475d for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @jrbosch!

**Pull request merged**